src/util/dom_style.js

```diff
@@ -35,7 +36,7 @@
   var parseEl = fabric.document.createElement('div'),
       supportsOpacity = typeof parseEl.style.opacity === 'string',
       supportsFilters = typeof parseEl.style.filter === 'string',
-      reOpacity = /alpha\s*\(\s*opacity\s*=\s*([^\)]+)\)/,
+      reOpacity = /alpha\s*\(\s*opacity\s*=\s*([^)]+)\)/,
```

Comment:
another lint caught thing

Reply:
It seems that linting now works for us

Reply:
it was working even before, but our lint config lacked lint-recommended plugin, and had just the old formatting rules.